migrations/20240505060955-add-index-for-order-pending-or-manual-filters.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.query(`
+      CREATE INDEX CONCURRENTLY IF NOT EXISTS "orders_isPendingContribution" ON "Orders" (((data #>> '{isPendingContribution}')::text)) where ((data #>> '{isPendingContribution}'::text[]) = 'true'::text);
+    `);
+
+    await queryInterface.sequelize.query(`
+      CREATE INDEX CONCURRENTLY IF NOT EXISTS "orders_isManualContribution" ON "Orders" (((data #>> '{isManualContribution}')::text)) where ((data #>> '{isManualContribution}'::text[]) = 'true'::text);
+    `);
```

Comment:
Not sure if we need those casts since the double `>>` already casts to text. Also, I think the second cast is wrong, it is not an array of texts.

If you need an example of how to index data fields:
https://github.com/opencollective/opencollective-api/pull/10037/files#diff-3af06b8af92da31930b6a5d8634d7dcca72dfe3d1703564986697d827cc82d78R8-R12

Reply:
I will double check the indexes with the example you provided. The `::text[]` cast refers to the argument of `#>>`, but its probably unnecessary, I just copied the database format after creating the index manually.